src/Desktop/ColorContrastAnalyzer/ColorContrastConfig.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Axe.Windows.Desktop.ColorContrastAnalyzer
+{
+    class ColorContrastConfig : IColorContrastConfig
+    {
+        internal const int DefaultMaxTextThickness = 20;
+        internal const int DefaultMinColorTransitions = 4;
+        internal const int DefaultMinSpaceBetweenSamples = 12;
+        internal const int DefaultTransitionCountDominanceFactor = 2;
+        internal const AnalyzerVersion DefaultAnalyzerVersion = AnalyzerVersion.V1;
+        internal const double DefaultHighConfidenceThreshold = 0.1;
+        internal const double DefaultMidConfidenceThreshold = 0.3;
+        internal const double DefaultLowConfidenceThreshold = 0.5;
```

Comment:
I flipped back and forth on whether it'd be more understandable to invert the confidence thresholds. At first I was confused that the HighConfidenceThreshold was lower than the MidConfidenceThreshold. But it made more sense as I dug deeper into V2's approach. If we want to change this, we could invert these and pass (1 - ConfidenceThreshold) to the underlying V2 calculator. I am fine either way.

Reply:
I waffled on this, partly because it felt like I was exposing too much of the implementation in the names. After taking another look, intuitive seems good. I've update the constants and code so that High confidence defaults to 90%, Mid to 70%, and Low to 50%.